manifests/Devolutions/2020.1.20.0.yaml

```
@@ -0,0 +1,21 @@
+Id: Devolutions.RemoteDesktopManagerFree
+Name: Remote Desktop Manager Free
```

```suggestion
Name: RemoteDesktopManagerFree
```
I think you should move the yaml file into the `manifests/Devolutions/RemoteDesktopManagerFree` folder and change the name field to `RemoteDesktopManagerFree` in order to pass the validation test.

---

The name and publisher fields are allowed to be different from the package name, the validation only checks if the ID matches the folder structure.